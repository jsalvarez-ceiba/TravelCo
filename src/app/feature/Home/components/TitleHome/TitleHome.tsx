import React from 'react';

const TitleHome = () => <h1> TravelCo </h1>;

export default TitleHome;
