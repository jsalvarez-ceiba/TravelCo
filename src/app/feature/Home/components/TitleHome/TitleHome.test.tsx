import React from 'react';
import renderer from 'react-test-renderer';
import TitleHome from './TitleHome';
import { mount } from 'enzyme';

interface Action {
    type:string
}

describe('Pruebas al componente TitleHome', () => {

    const Title = renderer.create(<TitleHome />).toJSON();

    test('Snapshot Component', () => {
        expect(Title).toMatchSnapshot();
    });
})



