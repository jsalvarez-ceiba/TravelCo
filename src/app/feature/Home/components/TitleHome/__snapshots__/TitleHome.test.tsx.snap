// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas al componente TitleHome Snapshot Component 1`] = `
<h1>
   TravelCo 
</h1>
`;
